src/main/java/com/mate/fruitshop/service/impl/ReportCreatorServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.mate.fruitshop.service.impl;
+
+import com.mate.fruitshop.dao.FruitStorageDao;
+import com.mate.fruitshop.dao.impl.FruitStorageDaoImpl;
+import com.mate.fruitshop.model.FruitEntry;
+import com.mate.fruitshop.service.ReportCreatorService;
+import java.util.List;
+
+public class ReportCreatorServiceImpl implements ReportCreatorService {
+
+    public static final String REPORT_HEADER = "fruit,quantity";
+    public static final String DELIMITER = ",";
+
+    @Override
+    public String createReport() {
+        FruitStorageDao dao = new FruitStorageDaoImpl();
+        List<FruitEntry> entries = dao.getAllFruitEntries();
+        StringBuilder strBuilder = new StringBuilder();
+        strBuilder.append(REPORT_HEADER).append(System.lineSeparator());
+        for (FruitEntry entry : entries) {
+            strBuilder.append(entry.getFruitName())
+                    .append(DELIMITER)
+                    .append(entry.getQuantity())

```

Comment:
let's create method `toString()` or `print()` inside the `FruitEntry` class, that will create the output for one fruit in needed format.  This will do code more simple and readable.